### ISSUE TITLE
Some icons are missing in UI in 2.10

### ISSUE THREAD

**Issue by idzikovsky** [snapcrafters/gimp#24]
![image](https://user-images.githubusercontent.com/23353239/39936424-1ba41f28-5555-11e8-8d3a-0ea5046864f2.png)

**Comment by diddledan** [comment]
Yes, I have struggled with this, and seemingly cannot get the stock images to be found. I can fix it by using core18 but that isn't a supported/supportable scenario yet.
